build/bootstrap/bootstrap-debian.sh

```diff
@@ -57,9 +51,15 @@ sha256sum -c - <<EOF
 EOF
 sudo tar -C /usr/local -zxf /tmp/go.tgz && rm /tmp/go.tgz
 
-# Clone CockroachDB.
-git clone https://github.com/cockroachdb/cockroach "$(go env GOPATH)/src/github.com/cockroachdb/cockroach"
-git -C "$(go env GOPATH)/src/github.com/cockroachdb/cockroach" submodule update --init
+# Install Docker compose.
+sudo curl -L "https://github.com/docker/compose/releases/latest/download/docker-compose-linux-x86_64" -o /usr/local/bin/docker-compose
```

Comment:
Let's either revert this to a pinned version or switch to installing `docker-compose-v2` with `apt-get`. We want to always use an exact, known version. This makes running the same command multiple times do the same thing (vs. having something change out from under you from one run to the next).

Reply:
> switch to installing docker-compose-v2 with apt-get
this installs `compose` as a plugin rather than standalone. I am switching to pinned version.

Reply:
@jlinder updated the PR. PTAL.